customizations/README.md

```diff
@@ -1,122 +1,107 @@
+   - [Configure TLS termination](#configure-tls-termination-in-the-service)
+3. [Use external Databases](#use-external-databases). Our recommendation is to use managed DBs with backups and so on.
+
+Optional configurations:
 
-Configuring it is optional:
 - [Configure scale of the components](#components-scaling)
 - Use your own bucket to store the data (By default, GCP CARTO buckets are used)
 
-## Architecture diagram
+## How to apply the configurations
 
-<!--
-TODO: We should add an arquitectural diagram to make it easier for customers to understand the parts and the relationship between them.
--->
+Create a dedicated [yaml](https://yaml.org/) file `client-conf.yaml` for your configuration. For example, you could create a file with the next content:
 
-## How to define customizations
-
-There are two ways to configure or customize the deployment:
-- [**RECOMMENDED**] Create a dedicated `customization.yaml` ([yaml official documentation](https://yaml.org/)) file. For example, you can create a file with the next content:
   ```yaml
   appConfigValues:
     selfHostedDomain: "my.domain.com"
+  ```
 
-  appSecrets:
-    googleMapsApiKey:
-      value: "<google-maps-api-key>"
+And add the following at the end of ALL the `helm install` or `helm upgrade` command:
 
-  router:
-    service:
-      type: LoadBalancer
-  ```
-  And add the following at the end of ALL the install or upgrade command:
-  ```bash
-  ... -f customization.yaml
-  ```
-- Use the parameters as arguments. You can specify each parameter using the `--set key=value[,key=value]` argument. For example, add the following at the end of ALL the install or upgrade command:
   ```bash
-  ... --set appConfigValues.selfHostedDomain=my.domain.com \
-    --set appSecrets.googleMapsApiKey.value=<google-maps-api-key> \
-    --set router.service.type=LoadBalancer
+  helm instal .. -f client-conf.yaml
   ```
 
+You can also override values through the command-line to `helm`. Adding the argument: `--set key=value[,key=value]`
+
+## Available Configurations
 
-## Configure the domain of your self-hosted
+There are several things that you can configure in you CARTO Self Hosted:
 
-The most important step to have your CARTO self-hosted ready to be used is to configure the domain to be used.
+### Configure the domain of your Self Hosted
 
-> ⚠️ CARTO self-hosted is not designed to be used in the path of a URL, it needs a full domain or subdomain. ⚠️
+The most important step to have your CARTO Self Hosted ready to be used is to configure the domain to be used.
+
+> ⚠️ CARTO Self Hosted is not designed to be used in the path of a URL, it needs a full domain or subdomain. ⚠️
+
+To do this you need to [add the following customization](#how-to-apply-the-configurations):
 
-To do this you need to [add the following customization](#how-to-define-customizations):
 ```yaml
 appConfigValues:
   selfHostedDomain: "my.domain.com"
+appSecrets:
```

Comment:
you delete that from the example but add it in the section of define the domain?

maybe you missleading the section trying to fix my comments?